crates/parcel_config/src/parcel_rc_config.rs

```diff
@@ -0,0 +1,781 @@
+use std::path::Path;
+use std::path::PathBuf;
+
+use package_manager::PackageManager;
+use parcel_filesystem::FileSystem;
+use pathdiff::diff_paths;
+
+use super::config_error::ConfigError;
+use super::parcel_config::ParcelConfig;
+use super::parcel_config::PluginNode;
+use super::parcel_rc::Extends;
+use super::parcel_rc::ParcelRcFile;
+use super::partial_parcel_config::PartialParcelConfig;
+
+#[derive(Default)]
+pub struct LoadConfigOptions<'a> {
+  pub additional_reporters: Vec<PluginNode>,
+  pub config: Option<&'a str>,
+  pub fallback_config: Option<&'a str>,
+}
+
+/// Loads and validates .parcel_rc config
+pub struct ParcelRcConfig<'a, T, U> {
```

Comment:
I suppose this is a "parcelrc config loader"?

Reply:
Yeah, I can rename it